cloudnet/src/main/java/de/dytanic/cloudnet/command/commands/CommandTasks.java

```diff
@@ -277,6 +277,15 @@ private static void handleTaskSetCommands(SubCommandBuilder builder) {
                         exactStringIgnoreCase("staticServices"),
                         bool("staticServices")
                 )
+                .generateCommand(
```

Comment:
Same as above, the required permission is not a part of the cloud itself and shouldn't be in the cloud code.

Reply:
moved that into a "rewritten" version of the bridge command, mostly copied it tbh